articles/00-DownloadingTheXRTK.md

```diff
@@ -0,0 +1,69 @@
+# How to download the Mixed Reality Toolkit
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/logo.png)
```

Comment:
Do we really wanna put this logo at the top of every page?

Reply:
No, we want the NEW logo at the top of each page, which we'll replace once you have it!

Reply:
I asked him for the final assets but he hasn't delivered yet.